src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/AddressEndpointTest.java

```diff
@@ -200,11 +204,17 @@ private void assertOk(String url) throws Exception {
     address1.setUprn(UPRN1);
     address1.setFormattedAddress(FORMATTED_ADDRESS1);
     address1.setWelshFormattedAddress(WELSH_FORMATTED_ADDRESS1);
+    address1.setRegion(COUNTRY_CODE);
```

Comment:
The country code from AIMS will be "E1234567" ?
In which case we need to only set region to be first char.

Reply:
Will strip the first character if you agree with AI we are definitely getting the whole code.

I thought they were doing the processing to turn it into the representation we required, just giving us the first character, which is why I thought countryCode was a descriptive name. Have had discussions with Richard and given him the JSON message we would expect from them. He replied nothing can be guaranteed, including key names in the JSON results which were given in their backlog ticket, and many decisions still need to be taken around Address Index which may have an effect. There may well be rework, the lack of design clarity around service interfaces and generally, means we are building on jelly and moving forward in an agile way accepting change in inevitable is the only way to go.